lib/web_ui/dev/felt

```diff
@@ -122,3 +122,10 @@ else
 
   $DART_SDK_DIR/bin/dart --packages="$WEB_UI_DIR/.packages" "$SNAPSHOT_PATH" $@
 fi
+
+# Finally run the web_engine_analysis.sh to warn the developers against
+# analysis issues.
+# This would show the analysis errors during development time, and will save
+# LUCI resources.
+echo "Running analysis checks."
+web_engine_analysis.sh
```

Comment:
Can we do this in `felt analyze`? As written, it will run analysis after every command, which means that if you run multiple commands in a row, e.g. on CI, you'll run analysis repeatedly multiple times. That will slow down the dev cycle. Also, frequently I run tests on unfinished code that contains analysis warnings. Showing analysis warnings would just add noise to test results.